src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { movies } from './MoviesGrid';
+
+let ErrorBoundary = (props: any) => {
+  const isDataLoaded = !!movies.length;
```

Comment:
As movies are constans, `isDataloaded~ will always return true.
It you want to add loading fallback that shows something before the data is loaded.

The intention of ErrorBoundary is slightly different:
It's helpful when some code down the tree throws and error. Without error boundary the whole app will break, but with error boundary only children of ErrorBoundary will be unmounted.
There's a good example of how to implement ErrorBoundary: [link](https://reactjs.org/docs/error-boundaries.html)

Reply:
Fixed.